ott/problems/linear/potentials.py

```diff
@@ -213,14 +206,30 @@ def callback(x: jnp.ndarray) -> float:
       # see proof of Prop. 2 in https://arxiv.org/pdf/2109.12004.pdf
       potential = self._f
       y = self._geom.x
-      prob_weights = self._a
+      prob_weights = self.a
     else:
       potential = self._g
       y = self._geom.y
-      prob_weights = self._b
+      prob_weights = self.b
 
     return callback
 
+  @property
```

Comment:
I think all of what's below should be removed.

Reply:
Done.